app.js

```diff
@@ -48,6 +51,7 @@ var tableEl = document.getElementById('stores');
   for (obj in cookieStores) {
```

Comment:
There's a lot of stuff happening in this `Store.render()` method. I would recommend breaking this out into some smaller methods so you can encapsulate one specific piece of functionality, and then call that method when and where you need it. It also help with knowing exactly what that one little method is doing.